components/plurals/src/rules/reference/ast.rs

```diff
@@ -86,6 +86,7 @@ use core::ops::RangeInclusive;
 /// [`AndConditions`]: AndCondition
 #[derive(Debug, Clone, PartialEq)]
 #[allow(missing_docs)] // TODO(#1029) - Add missing docs.
+#[allow(clippy::exhaustive_structs)] // this type is stable
```

Comment:
I don't have strong opinions on the ones in this file, except to note that this is "reference" code, so unlikely to be used by a lot of people. I think I lean toward making these stable, because code that _does_ use this module will probably want to construct these and handle all cases exhaustively.